data/sql/updates/pending_db_characters/glicko2.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE `character_battleground_rating` (
+  `guid` INT UNSIGNED NOT NULL COMMENT 'Character GUID',
```

Comment:
I don't have the link handy, but there is a wiki page detailing style for DB naming. Mostly capitalization on column names is what I'm thinking of.